subsys/dm/timeslot_queue.c

```diff
@@ -98,7 +98,6 @@ int timeslot_queue_append(struct dm_request *req, uint32_t start_ref_tick)
 	}
 
 	item->timeslot_req.start_time = start_time;
-	req->access_address++;
```

Comment:
maybe I'm missing something here, but where are you creating a new seed for a follow-up ranging? I think the seed should be different for each ranging.

Reply:
The creation a new seed is implemented in the sample code. It is done after each synchronization.
The advertiser puts it into the `manufacture data` structure and updates the advertising data to pass it to he scanner. This way both devices have the same seed value.

Reply:
Yes but that's only for newly synced devices, right?
I think we should update the random value each ranging, meaning also for a follow-up ranging where we schedule a new timeslot without syncronising again. We could do that for example whenever we get MPSL_TIMESLOT_SIGNAL_SESSION_IDLE, MPSL_TIMESLOT_SIGNAL_SESSION_BLOCKED or MPSL_TIMESLOT_SIGNAL_CANCELLED.

Reply:
You are right when `CONFIG_DM_TIMESLOT_RESCHEDULE` is enabled, there will be a new timestlot scheduled, then the seed should be updated.